tests/time_encoding.go

```diff
@@ -0,0 +1,137 @@
+// Code generated by fastssz. DO NOT EDIT.
+// Hash: ddcde9be27549f8b7a4ce0f961180e583140e521dfde43f8328aaeab91449742
+package tests
+
+import (
+	ssz "github.com/ferranbt/fastssz"
+)
+
+// MarshalSSZ ssz marshals the TimeType object
+func (t *TimeType) MarshalSSZ() ([]byte, error) {
+	return ssz.MarshalSSZ(t)
+}
+
+// MarshalSSZTo ssz marshals the TimeType object to a target array
+func (t *TimeType) MarshalSSZTo(buf []byte) (dst []byte, err error) {
+	dst = buf
+
+	// Field (0) 'Timestamp'
+	dst = ssz.MarshalUint64(dst, uint64(t.Timestamp.Unix()))
+
+	// Field (1) 'Int'
+	dst = ssz.MarshalUint64(dst, t.Int)
+
+	return
+}
+
+// UnmarshalSSZ ssz unmarshals the TimeType object
+func (t *TimeType) UnmarshalSSZ(buf []byte) error {
+	var err error
+	size := uint64(len(buf))
+	if size != 16 {
+		return ssz.ErrSize
+	}
+
+	// Field (0) 'Timestamp'
+	t.Timestamp = time.Unix(int64(ssz.UnmarshallUint64(buf[0:8])), 0).UTC()
```

Comment:
You need to add an entry [here](https://github.com/ferranbt/fastssz/blob/bd3f11aa9eaa37e3995d5eaf80ea994dc9ae20c2/sszgen/generator/generator.go#L454) to resolve the import.

Reply:
Added.